File: source/code/vbnet/authentication/login_attempt_req_expand_account.vb
Dim loginRequest = New UsernamePasswordRequestBuilder() _
    .SetUsernameOrEmail("han@newrepublic.gov") _
    .SetPassword("first2shoot!") _
    .Build()

Dim loginResult = Await app.AuthenticateAccountAsync(
    loginRequest,
    Function(opt) opt.Expand(Function(result) result.GetAccount()))

File: source/code/java/authentication/login_attempt_req_expand_account.java
import com.stormpath.sdk.authc;
import com.stormpath.sdk.account;

try {
  AuthenticationRequest authenticationRequest = UsernamePasswordRequest.builder()
              .setUsernameOrEmail("han@newrepublic.gov")
              .setPassword("first2shoot!")
              .withResponseOptions(new DefaultBasicAuthenticationOptions().withAccount())
              .build();
  AuthenticationResult result = application.authenticateAccount(authenticationRequest);

  Account account = result.getAccount();
} catch(IncorrectCredentialsException e) {
  e.printStackTrace();
}
